app/api/routes/helpers.py

```diff
@@ -11,7 +11,7 @@
 def unauthorized_response():
     message = "The email or password you submitted is incorrect " \
               "or your account is not allowed api access"
-    payload = {'errors': {"invalid-credentials": {"message": message}}}
+    payload = {'errors': {"unauthorized": {"message": message}}}
```

Comment:
The only status code that returns an "invalid-*" response is a 422. For all the rest of the routes, the error is the same as the status, which in the case of 401 is "Unauthorized". I feel this API change is necessary for consistency